modules/mobile/app/models/mobile/v0/contracts/get_paginated_list.rb

```diff
@@ -7,10 +7,11 @@ class GetPaginatedList < Base
         params do
           optional(:start_date).maybe(:date_time, :filled?)
           optional(:end_date).maybe(:date_time, :filled?)
```

Comment:
Unclear how we we want to handle this, but this is one approach. AFAICT, we're currently treating page number and page size as required because the existing pagination approach would break if they were missing. We can handle this a few ways. We can do some combination of 1) change the pagination code to work without them 2) raise an error in the paginator if they're missing 3) require them here on the assumption that we'll always use this for the paginator. I'm open to suggestion here.

Reply:
I think these should remain optional and that we should do 1) change the pagination code to work without them. Otherwise I think we'd have to version appointments and claims too because the contract changed (even though we're fixing the code to not to break if they're missing).